src/core/thread.d

```diff
@@ -3886,12 +3890,13 @@ class Fiber
      *
      * Params:
      *  dg = The fiber function.
+     *  ce = If true, exceptions will be caught and assigned to m_unhandled
      *  sz = The stack size for this fiber.
      *
      * In:
      *  dg must not be null.
      */
-    this( void delegate() dg, size_t sz = PAGESIZE*4 ) nothrow
+    this( void delegate() dg, bool ce = true, size_t sz = PAGESIZE*4 ) nothrow
```

Comment:
As an alternative I could also add two new c'tors with the same arguments as this one right now